R/source.R

```diff
@@ -83,11 +87,28 @@ activate <- function(head_branch = gert::git_branch(),
                        "touchstone.default_base_branch",
                        "main"
                      ),
+                     n = 1,
                      env = parent.frame()) {
+  if (!rlang::is_interactive()) {
+    if (Sys.getenv("GITHUB_ACTIONS") == TRUE) {
+      cat(paste0(
+        "::warning ::activate() is meant for interactive use ",
+        "only, make sure script.R works as intended!"
+      ))
+    } else {
+      cli::cli_warn(paste0(
+        "{.fun activate} is meant for interactive use",
+        "only, make sure {.file script.R} works as intended!"
+      ))
+    }
+  }
+
+
   suppressMessages({
     withr::local_envvar(
       GITHUB_BASE_REF = base_branch,
       GITHUB_HEAD_REF = head_branch,
+      touchstone.n_runs = n,
```

Comment:
Just to remain consistent here, I think this should be an R option, not an env variable.